examples/stubbing-spying__intercept/cypress/integration/stubbed-api-spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="Cypress" />
+
+// in this test, there is no backend for the app to hit
+// @see https://github.com/cypress-io/cypress/issues/9599
```

Comment:
This issue was closed, does this still need to be referenced?

Reply:
I think yes, because it was causing a problem (because the real server used to reply to the OPTIONS request), so I would keep the URL around